ios/Podfile.lock

```diff
@@ -190,11 +190,11 @@ PODS:
   - nanopb/encode (2.30908.0)
   - OpenSSL-Universal (1.1.1100)
```

Comment:
unnecessary but i think it would be cool if the error sheets auto-dismissed in some cases. like when i click "review tx" and the unlock sheet pops up, it should go down automatically if i unlock the device